test_data/makedata_suites/015_cluster_wait.js

```diff
@@ -0,0 +1,57 @@
+/* global print, db, internal, arango */
```

Comment:
please make sure you run jslint from the arangodb repo over this:
```
 ./utils/jslint.sh ../release-test-automation/test_data/makedata_suites/
```
at least the files you add should be good - we need to work on the others.